scout/server/blueprints/cases/templates/cases/gene_panel.html

```diff
@@ -18,7 +18,7 @@
                   {{ panel.display_name|truncate(30, True) }}
                 </a>
                 {% if panel.is_default %}
-                  <span class="badge badgel-default pull-right">Default</span>
```

Comment:
🥯

Reply:
Yes, or maybe schampoo of some sort? Could also be something from the lab that went wrong!